### PR TITLE
Serve CSV export files as medias with Nginx

### PR DESCRIPTION
**Pull request by rmoch**
## Purpose

Previous PR regarding generating and serving CSV export files was incomplete.
We now make Nginx to serve those files after their access has been verified by Fonzie ACL view

Mount previously created `export` volume in Nginx container, and set related Nginx configuration

### CODE REVIEW

File: apps/edxapp/templates/nginx/_configs/lms.conf.j2 [hunk 1]
```diff
@@ -58,4 +58,11 @@ server {
     root /data/static;
     try_files /$file =404;
   }
+
+  location ~ ^/export/(?P<file>.*) {
+    root /data/static;
```
Comment (jmaupetit) [inline apps/edxapp/templates/nginx/_configs/lms.conf.j2]:
```suggestion
    root /data/export;
```

File: apps/edxapp/templates/nginx/_configs/lms.conf.j2 [hunk 2]
```diff
@@ -58,4 +58,11 @@ server {
...
+    try_files /$file =404;
+  }
+
+
```
Comment (jmaupetit) [inline apps/edxapp/templates/nginx/_configs/lms.conf.j2]: You should modify the CMS's ConfigMap, not the LMS.

File: apps/edxapp/templates/nginx/dc.yml.j2 [hunk 3]
```diff
@@ -48,6 +48,9 @@ spec:
             - mountPath: /data/static
               name: edxapp-v-static
               readOnly: true
+            - mountPath: /data/export
+              name: edxapp-v-export
+              readOnly: true
```
Comment (jmaupetit) [inline apps/edxapp/templates/nginx/dc.yml.j2]: You've forgot the PVC definition below 👇

**Review by jmaupetit — COMMENTED** [review]
Don't forget to edit the CHANGELOG!

File: apps/edxapp/templates/nginx/_configs/lms.conf.j2 (hunk 2 above)
Comment (jmaupetit) [reply apps/edxapp/templates/nginx/_configs/lms.conf.j2]: Forget about this comment. I was mistaken.

File: CHANGELOG.md [hunk 4]
```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Changed
 
 - Increase length of secret keys to 50 characters
+- Mount previously added `export` volume to nginx and add configuration
+  to serve it.
```
Comment (jmaupetit) [inline CHANGELOG.md:18]:
```suggestion
  - Rename the edxapp `exports` volume to `export`
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
```
Comment (jmaupetit) [inline CHANGELOG.md:20]: Add a `fixed` subsection and add:

```markdown
### Fixed

- Mount edxapp `exports` volume in `nginx` configuration to serve exported data
```

File: apps/edxapp/templates/_volumes/export.yml.j2 [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+  labels:
+    app: edxapp
+    version: "{{ edxapp_image_tag }}"
+    deployment_stamp: "{{ deployment_stamp }}"
+spec:
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: {{ edxapp_export_volume_size }}
```
Comment (jmaupetit) [inline apps/edxapp/templates/_volumes/export.yml.j2]: You should also remove the `exports.yml.j2` volume.

Comment (rmoch) [reply apps/edxapp/templates/_volumes/export.yml.j2]: 👍

File: CHANGELOG.md (hunk 5 above)
Comment (rmoch) [reply CHANGELOG.md]: I do not get why this.

**Review by jmaupetit — APPROVED** [review]

**Pull request merged**